lib/internal/Magento/Framework/Stdlib/DateTime/Timezone.php

```diff
@@ -85,23 +85,31 @@ public function __construct(
     }
 
     /**
-     * {@inheritdoc}
+     * Return path to default timezone
```

Comment:
You shouldn't duplicate descriptions in the implementation. They are in the interface. Please return them to `{@inheritdoc}`

Reply:
Thank you for the explanation. So should I just ignore these warnings then?
https://travis-ci.org/magento/magento2/jobs/429304963

Reply:
Basically you could ignore those since {@inheritdoc} may be used in such cases and there's a bug in our sniffer. But it would be good if you make changes according to this guide https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#inheritdoc. As you could see there should be enough to remove curly braces {}